app/controllers/api/reservations_controller.rb

```diff
@@ -8,9 +8,8 @@ def index
     end
 
     def create
-      reserve = Reservation.new(start_date: params[:start_date], city: params[:city])
+      reserve = Reservation.new(reserve_param)
       @car = Car.find(params[:car_id])

```

Comment:
delete this ligne of code ise useless when you handle them inside reserve_param function.

Reply:
delete this ligne of code ise useless when you handle them inside reserve_param function.

Reply:
Thanks for the feed back but this how the URL for post request  would look like in frontend, which means we can get the id using find params
`http://localhost:3000/api/cars/{id}/reservations`

Reply:
this routes `resources :cars , only:[:index] do
resources reservations[:index, :create]`

was used to get this

`http://localhost:3000/api/cars/{id}/reservations`

I have used some api with routes this way on the frontend.

but i have made the changes as requested